docs/pages/guides/monorepos.mdx

```diff
@@ -230,6 +231,52 @@ export default function App() {
 }
 ```
 
+## Using EAS Workflows with monorepos
+
+[EAS Workflows](/eas/workflows/introduction/) automate your CI/CD processes, such as creating builds, publishing updates, and running tests. When working with monorepos, each app can have its own workflows.
+
+The **.eas/workflows** directory lives in the same directory as **eas.json**. If you have multiple Expo apps in your monorepo, each app directory will have its own **eas.json** and **.eas/workflows** directory.
+
+For example:
+
+<FileTree
+  files={[
+    ['apps/cool-app/.eas/workflows/build.yml', ''],
+    ['apps/cool-app/eas.json', ''],
+    ['apps/another-app/.eas/workflows/deploy.yml', ''],
+    ['apps/another-app/eas.json', ''],
+  ]}
+/>
+
+### Connect GitHub repository
```

Comment:
This section has more details and feels more complete than the one in EAS Workflows setup guide: https://github.com/expo/expo/pull/40719/files#diff-5833e89235cfda80e894cb96db6f091a4dee921e857782be55d1959b346a9746R139. 

I think we should have a single source of truth that provides all details to know (possibly in EAS Workflows setup guide) and then link it to this guide here.